tests/llm_cache.py

```diff
@@ -33,6 +35,8 @@ def __init__(self, teardown_method: Callable[[T], None] | None = None):
         else:
             self._teardown = teardown_method
 
+        self._preexisting_max_tkv = os.getenv("VLLM_DT_MAX_BATCH_TKV_LIMIT")
```

Comment:
Tests that modify `VLLM_DT_MAX_BATCH_TKV_LIMIT` should be using `monkeypatch` which would automatically restore the value after the test. Can we use that instead of having a special case reset here?